modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js

```diff
@@ -68,6 +70,37 @@ class DDMFormHandler {
 			}
 		});
 	}
+
+	checkCPInstanceOptions() {
+		var emptyRequiredField = false;
+		const addToCartButton = document.querySelector('.btn-add-to-cart');
+		const ddmFormValues = JSON.stringify(this.fields);
+
+		for (const ddmFormValue of JSON.parse(ddmFormValues)) {
```

Comment:
There's no need to stringify `this.fields` and parse it again. 

`for (const ddmFormValue of JSON.parse(this.fields)) {`